### ISSUE TITLE
A question about textual deviation score.

### ISSUE THREAD

**Issue by SydCS** [ml-stat-Sustech/CLIP_Calibration#1]
Hello. Thank you for this work!

I have a question about the method: 
Each class has its different textual deviation score. 
After multiplying it on the CLIP logits, might it influence (undermine/increase) the classification accuracy?

Looking forward to your kind reply!

**Comment by Claydon-Wang** [comment]
Hi there, 

Thanks for your interest! Similar to post-hoc methods like temperature scaling, DAC only adjusts the output confidence and does not affect the classification accuracy.

Best,
Shuoyuan

**Comment by SydCS** [comment]
I see!

Since the framework is a bit deep. Could you tell me which base_calibrator is used to calibrate the logits based on TD score?
[vl_calibrator](https://github.com/ml-stat-Sustech/CLIP_Calibration/blob/main/trainers/calibration/vl_calibrator.py#L83)

**Comment by Claydon-Wang** [comment]
Hi there,

`base_calibrator` can be split into scaling-based method like [tempscaling](https://github.com/ml-stat-Sustech/CLIP_Calibration/blob/main/trainers/calibration/tempscaling.py) and bin-based method like [isotonic regression](https://github.com/ml-stat-Sustech/CLIP_Calibration/blob/main/trainers/calibration/multi_isotonic_regression.py)
For more detailed implementation, you can fine the corresponding function in [build_base_calibrator](https://github.com/ml-stat-Sustech/CLIP_Calibration/blob/main/trainers/calibration/vl_calibrator.py#L112). For the performance reported in Table 1, we set`base_calibrator=None`.

Best,
Shuoyuan

**Comment by SydCS** [comment]
Got it! The raw result comes from using dac_calibrator only.
Thank you for these responses!
